collector/collector.go

```diff
@@ -87,7 +87,7 @@ func NewCollector(factories component.Factories) *Collector {
 func (c *Collector) Start(ctx context.Context) error {
 	params := service.CollectorSettings{
 		BuildInfo: component.BuildInfo{
-			Command:     "otelcol",
+			Command:     "otelcol-lambda",
```

Comment:
probably good to do anyway if you wanted to move this out of this PR

Reply:
Yeah, I'll make a separate PR for this.  Didn't even intend to leave it here.

Reply:
#314 fixes this.